createNPMPackage.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+yarn install --immutable
+
+if ! CURRENT_VERSION=$(node scripts/set-version.js); then
+  exit 1
+fi
+
+yarn build
```

Comment:
I don't think we even have `build` script here in the repo. We use `yarn prepare`.